packages/pg-v5/commands/pull.js

```diff
@@ -75,11 +57,27 @@ const prepare = co.wrap(function * (target) {
   }
 })
 
-function connstring (uri, skipDFlag) {
-  let user = uri.user ? `-U ${uri.user}` : ''
-  let host = uri.host ? `-h ${uri.host}` : ''
-  let port = uri.port ? `-p ${uri.port}` : ''
-  return `${user} ${host} ${port} ${skipDFlag ? '' : '-d'} ${uri.database}`
+function connArgs (uri, skipDFlag) {
+  const args = []
+
+  if (uri.user) {
+    args.push('-U')
```

Comment:
no need to make these separate calls. There is also a lot of unnecessary interpolation here.

```
if (uri.user) args.push('-U', uri.user)
```

Reply:
Great catch! @jdxcode 💯